aws-android-sdk-core/src/main/java/com/amazonaws/logging/ApacheCommonsLogging.java

```diff
@@ -15,14 +15,14 @@
 
 package com.amazonaws.logging;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
```

Comment:
Oh man, I see. This was shadowing the package local copy? And this document still compiles, because now `LogFactory` refers to the package local class? Creepy stuff, sigh.